### PR TITLE
Add EKS terraform definitions

### PR DESCRIPTION
**Pull request by smaslennikov**
works or something

### CODE REVIEW

**Comment by naftulikay** [comment]
If you can please sandbox to `examples/gke/*` that would be rad :+1:

File: terraform/eks/main.tf [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+}
+
+resource "aws_iam_role_policy_attachment" "eks-node-AmazonEC2ContainerRegistryReadOnly" {
+  policy_arn = "arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly"
+  role       = "${aws_iam_role.eks-node.name}"
+}
+
+# Allow nodes and cluster to play with route53
+
+resource "aws_iam_policy" "nodes-r53-policy" {
```
Comment (naftulikay) [inline terraform/eks/main.tf]: mother of god

File: terraform/eks/main.tf [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+  most_recent = true
+  owners      = ["602401143452"] # Amazon
+}
+
+# EKS currently documents this required userdata for EKS worker nodes to
+# properly configure Kubernetes applications on the EC2 instance.
+# We utilize a Terraform local here to simplify Base64 encoding this
+# information into the AutoScaling Launch Configuration.
+# More information: https://amazon-eks.s3-us-west-2.amazonaws.com/1.10.3/2018-06-05/amazon-eks-nodegroup.yaml
+locals {
```
Comment (naftulikay) [inline terraform/eks/main.tf]: father of dog

Comment (smaslennikov) [reply terraform/eks/main.tf]: yes my son

**Pull request merged**